scripts/ocr_runner.py

```
@@ -0,0 +1,198 @@
+                orig_len = orig_txt.stat().st_size if orig_txt.exists() else 0
+                if fb_chars > orig_len and fb_chars >= 200:
+                    # backup
+                    bk = ROOT / 'epstein_project' / 'text_pre_ocr_fix'
+                    bk.mkdir(parents=True, exist_ok=True)
+                    if orig_txt.exists():
+                        shutil.copy2(orig_txt, bk / orig_txt.name)
+                    orig_txt.write_text(fb_txt.read_text(encoding='utf-8'), encoding='utf-8')
+                    rec['replaced_text'] = True
+            except subprocess.TimeoutExpired:
+                rec['fallback_status'] = 'fallback_timeout'
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)